src/main/java/seedu/guest/model/guest/Request.java

```diff
@@ -0,0 +1,68 @@
+package seedu.guest.model.guest;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.guest.commons.util.AppUtil.checkArgument;
+
+/**
+ * The class Requests.
```

Comment:
Should this be Request? Also, should we follow the same documentation-style for the other field classes as well (e.g Address, Name, etc...)?

Reply:
changed